models/bamf_nnunet_mr_liver/utils/BamfProcessorRunner.py

```diff
@@ -0,0 +1,66 @@
+"""
+-------------------------------------------------
+MHub - Run Module for ensembling nnUNet inference.
+-------------------------------------------------
+-------------------------------------------------
+Author: Rahul Soni
+Email:  rahul.soni@bamfhealth.com
+-------------------------------------------------
+"""
+
+from mhubio.core import Instance, InstanceData, DataType, FileType, CT, SEG
+from mhubio.core import Module, IO
+import os, numpy as np
+import SimpleITK as sitk
+from skimage import measure, filters
+import numpy as np
+import shutil
+
+
+
+class BamfProcessorRunner(Module):
+
+    @IO.Instance
+    @IO.Input('in_data', 'nifti:mod=ct|mr', the='input data to run nnunet on')
+    @IO.Output('out_data', 'bamf_processed.nrrd', 'nrrd:mod=seg:processor=bamf', data='in_data', the="keep the two largest connected components of the segmentation and remove all other ones")
+    def task(self, instance: Instance, in_data: InstanceData, out_data: InstanceData) -> None:
+       # Log bamf runner info
```

Comment:
The comment needs one more space to match the indent right :D

Reply:
I'd add an empty line above for better readability, but that's a personal preference ;)